sources/api/netdog/src/cli/generate_net_config.rs

```diff
@@ -48,8 +50,16 @@ pub(crate) fn run() -> Result<()> {
     remove_old_primary_interface()?;
     write_primary_interface(&primary_interface)?;
 
-    let wicked_interfaces = net_config.as_wicked_interfaces();
-    for interface in wicked_interfaces {
+    let mut wicked_interfaces = net_config.as_wicked_interfaces();
+    for interface in &mut wicked_interfaces {
+        // The kernel command line is too limited to fully specify an interface's configuration;
+        // fix some defaults to match legacy behavior.
+        // Note: we only allow 1 interface to be listed via kernel command line, so this will only
+        // be added to a single interface
+        if from_cmd_line {
+            interface.accept_ra();
+        }
```

Comment:
I don't understand why this is conditional on the interface being configured via the kernel command line. Before this change, the interface's `accept_ra` sysctl would be set to `2` regardless. Should this instead be conditional on it being the primary interface? (With #2615 later it would also need to check for conflicting explicit configuration, but not yet.)

Reply:
Good question.  We want to make this fully configurable in #2615 .  

At the moment, metal deployments via EKS-A don't support IPv6, so limiting this change to variants that get their config via the command line (AWS/VMware) feels pretty safe from any regression.  AWS/VMware variants configure a single interface (the primary) via command line, so this config will only be applied to that primary interface.